robotPositioner/robotPositioner.h

```diff
@@ -0,0 +1,382 @@
+#include <vector>
+#include <list>
+#include <cstdio>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <chrono>
+#include <csignal>
+#include <thread>
+#include <algorithm>
+
+
+// BoB robotics includes
+#include "../vicon/capture_control.h"
+#include "../vicon/udp.h"
+#include "../robots/norbot.h"
+#include "../video/see3cam_cu40.h"
+#include "../imgproc/opencv_unwrap_360.h"
+
+using namespace BoBRobotics::Vicon;
+using namespace BoBRobotics::ImgProc;
+using namespace BoBRobotics::Video;
+using namespace std::literals;
+
+#define PI 3.14159265359
+
+class robotPositioner {
```

Comment:
I think this class does too much stuff! Things that definitely don't belong are:
- Vicon capture control
- Camera capture